src/MagnumPlugins/DrFlacAudioImporter/DrFlacImporter.cpp

```diff
@@ -74,12 +103,30 @@ void DrFlacImporter::doOpenData(Containers::ArrayView<const char> data) {
         return;
     }
 
-    Containers::Array<char> tempData(samples*sizeof(int32_t));
+    _frequency = handle->sampleRate;
+    _format = flacFormatTable[numChannels][normalizedBitsPerSample];
```

Comment:
Also, since `numChannels` and `normalizedBytesPerSample`  is always >= 1, I would do it like this:

```cpp
_format = flacFormatTable[numChannels - 1][normalizedBytesPerSample - 1];
```

and then remove the sentinel first column/row from the table.

Then I would also make unreachable entries (3 and 5 channels) of the table default-constructed (`Buffer::Format{}`) and then assert on them just to be sure:
```cpp
CORRADE_INTERNAL_ASSERT(_format != Buffer::Format{});
```